packages/terra-table/tests/nightwatch/multi-select-table/MultiSelectTableTestRoutes.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import React from 'react';
+import { Route } from 'react-router';
+import MultiSelectTableTests from './MultiSelectTableTests';
+import Table from './components/MultiRowSelectableTable';
+import TableOnChange from './components/MultiRowSelectableTableOnChange';
+import TablePreselected from './components/MultiRowSelectableTablePreselected';
+import TableNonSelectable from './components/MultiRowSelectableTableNonSelectable';
+import TableMaxCount from './components/MultiRowSelectableTableMaxCount';
+import TableMaxCountInitiallyMet from './components/MultiRowSelectableTableMaxCountInitiallyMet';
+import TableMaxCountInitiallyExceeded from './components/MultiRowSelectableTableMaxCountInitiallyExceeded';
+
+const routes = (
+  <div>
+    <Route path="/tests/multi-select-table-tests" component={MultiSelectTableTests} />
+    <Route path="/tests/multi-select-table-tests/default" component={Table} />
+    <Route path="/tests/multi-select-table-tests/onchange" component={TableOnChange} />
+    <Route path="/tests/multi-select-table-tests/preselected" component={TablePreselected} />
```

Comment:
The nice part of having the SelectableTableRows component it is handling the logic needed to wrap the table rows with the correct selected and selectable props. This being said, the preselected and non-selectable tests can be removed.

Reply:
It might be beneficial to leave them for several reasons. We know that the internal implementation of MultiSelectableRows uses the SelectableTableRows and there is a certain level of trust we can offload onto the SelectableTableRows. But it's still the responsibility/ acceptance criteria of the MultiSelectableRows component to have these behaviors. So these tests can be mentally viewed as an integration test of these two components.

In our Jest tests, we shallow render and assert against the props that MSR passes to SelectableTableRows. So we are offloading the responsibility to the contract of the SelectableRows component. But since it's shallow rendered, the actual behavior of the MSR component isn't actually asserted against. So Nightwatch tests cover that.

I'm fine with removing it, but perhaps it would be good for any consumer/wrapper around SelectableTableRows to have a base suite of tests like this to have and ensure the intended behavior is preserved/ any changes to the original behavior is called out.

Reply:
I see value in both sides. Whatever other's think.